src/python/pants/backend/BUILD

```diff
@@ -6,7 +6,7 @@ python_sources()
 
 # Setup rules about which parts of the codebase may depend on backend sources.
 
-DEFAULT_BACKEND_RULES = (
+DEFAULT_DEPENDENTS_RULES = (
```

Comment:
Sorry to be pedantic, but these aren't rules, right? a rule is the entire tuple. This is just the dependents part of the rule? So DEFAULT_ALLOWED_DEPENDENTS?

Reply:
If we're pedantic, the terms used in the docs are "rule set" consisting of one or more "selectors" followed by the "rules", so yes these are indeed the default rules for the rule sets below.

Reply:
Aha, to me a "rule" implies something that has both source and dest and action information. But whatever, this is fine for now.

Reply:
I can agree with that to some degree, but it was too atomic for me when documenting it--how do you name the parts that make up the rule? 

Maybe I was colored by too much implementation detail then..